sources/targets/Stride.Core.targets

```diff
@@ -46,12 +46,11 @@
         Condition="'$(StrideEnableCodeAnalysis)' != 'true'">
     <ItemGroup>
       <!-- We want to include Stride analyzers by default -->
-      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$([System.String]::Copy(%(FullPath)).Contains('Stride'))" />
+      <AnalyzersNotToRemove Include="@(Analyzer)" Condition="$(FullPath.Contains('Stride'))" />
```

Comment:
Revert that. It looks like you had a conflict but resolved it the wrong way.

Reply:
Right now the build is fairly fragile wrt those lines - let's wait until the PR is finalized and revert only then that way Noah can keep working on this.